.github/workflows/turing.yaml

```diff
@@ -476,7 +521,8 @@ jobs:
             --set turing.image.repository=${{ github.repository }}/turing-api \
             --set turing.image.tag=${{ env.TURING_API_VERSION }} \
             --set turing.config.RouterDefaults.Image=${{ env.LOCAL_REGISTRY }}/${{ github.repository }}/turing-router:${{ env.TURING_ROUTER_VERSION }} \
-            --values infra/e2e/turing.helm-values.yaml
+            --values infra/e2e/turing.helm-values.yaml \
+            --wait
```

Comment:
Since you made everything synchronous  (with `--wait` added to helm install commands), there is very little point in `wait-for-deployment` step. Not sure if we want to keep it

Reply:
Yeah I'll remove this. It's hard to tell which step it failed at if you do the wait for all to complete.